runtests.py

```diff
@@ -63,7 +69,11 @@
             # us skip creating migrations for the test models.
             'scribbler': 'scribbler.tests.migrations',
             'dayslog': 'dayslog.tests.migrations',
-        }
+        },
+        MEDIA_ROOT = '',
```

Comment:
Minor nit: these are inside the `settings.configure` call so they should not have spaces.